src/50_Sample_AMPs/Recipe.html

```diff
@@ -0,0 +1,466 @@
+          "width": 116,
+          "height": 41
+        }
+      },
+      "aggregateRating": {
+        "@type": "AggregateRating",
+        "ratingValue": "4.5",
+        "bestRating": "5",
+        "ratingCount": "3",
+        "reviewCount": "3"
+      },
+      "review": [{
+        "@type": "Review",
+        "name": "Yum!",
+        "reviewBody": "Words cannot express how delicious this tastes!",
+        "reviewRating": {
+          "@type": "Rating",
+          "ratingValue": "5"
+        },
+        "author": "Positive Patrick"
+      }, {
+        "@type": "Review",
+        "name": "Not bad, but prep is laborious",
+        "reviewBody": "Tastes really good, but cutting the silverskin takes a while. Try asking your butcher if they can do it, since they have specialized tools and skills!",
+        "reviewRating": {
+          "@type": "Rating",
+          "ratingValue": "4"
+        },
+        "author": "Negative Nancy"
+      }, {
+        "@type": "Review",
+        "name": "Unexpected side effects",
+        "reviewBody": "After taking it out of the oven, I left this dish to cool off next to my pet lizard, unfortunately now he's 350ft tall now and is currently destroying Tokyo, Japan.",
+        "reviewRating": {
+          "@type": "Rating",
+          "ratingValue": "4"
+        },
+        "author": "Lawrence Gonzalez"
+      }]
+    }
+  </script>
+
+
+  <!-- [Responsive `meta` tag](https://css-tricks.com/snippets/html/responsive-meta-tag/). -->
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+
+  <!-- This boilerplate style tag is required. -->
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+
+
+  <!-- Only one user-defined `style` tag is allowed. It must have an `amp-custom` attribute, and must be [under 50KB](https://www.ampproject.org/docs/get_started/technical_overview.html#all-css-must-be-inline-and-size-bound).
+
+   The `amp-carousel amp-img > img { object-fit: contain; }` rule preserves the image aspect ratios in the carousel. See [this GitHub issue](https://github.com/ampproject/amphtml/issues/1531#issuecomment-225461224) for potentially better methods recommended after this example was released.
```

Comment:
Don't explain the carousel -> explain what it does and reference the existing samples.